plugin/pkg/admission/exec/admission_test.go

```diff
@@ -146,20 +147,20 @@ func testAdmission(t *testing.T, pod *api.Pod, handler *DenyExec, shouldAccept b
 func TestDenyExecOnPrivileged(t *testing.T) {
 	privPod := validPod("privileged")
 	priv := true
-	privPod.Spec.Containers[0].SecurityContext = &api.SecurityContext{
+	privPod.Spec.Containers[0].SecurityContext = &corev1.SecurityContext{
 		Privileged: &priv,
 	}
 
 	hostPIDPod := validPod("hostPID")
-	hostPIDPod.Spec.SecurityContext = &api.PodSecurityContext{}
-	hostPIDPod.Spec.SecurityContext.HostPID = true
+	hostPIDPod.Spec.SecurityContext = &corev1.PodSecurityContext{}
+	hostPIDPod.Spec.HostPID = true
 
 	hostIPCPod := validPod("hostIPC")
-	hostIPCPod.Spec.SecurityContext = &api.PodSecurityContext{}
-	hostIPCPod.Spec.SecurityContext.HostIPC = true
+	hostIPCPod.Spec.SecurityContext = &corev1.PodSecurityContext{}
```

Comment:
nits: now we dont need this line, also for L155

Reply:
Addressed.

I have also removed L53 and L158, because they're very similar. Are those okay to be removed or I should revert changes for those 2?